data/json/snippets/zombie_anatomy.json

```diff
@@ -0,0 +1,89 @@
+      "A careless move dislodges some remaining air causing the throat of the undead monster to emit one more choked scream",
+      "The slack jaws and starved body of this creature make you think that it would have struggled to seriously harm anything on its own",
+      "Its distended lungs seem to be partially lined with a black and sticky mucus membrane"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_bone_general>",
+    "text": [
+      "Removing its bone plates proves more harduous than you would have thought, requiring you to almost put your entire weight on your tool to pry them off",
+      "Black liquid bubbles in the places where you managed to remove its skeletal growths",
+      "This one's armor falls off surprisingly easily, almost as if whatever puppets it wishes for it to lose the extra weight",
+      "The layout and segmentation of its bony carapace disturbingly resembles what you want to call plate mail, but its closer to the armor of an arthropod",
+      "These plates are almost impossible to remove until you realize they are linked to the internal skeleton with calcified pillars you must break before removing its armor"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_acid_general>",
+    "text": [
+      "The sheer pungent acrid odor of the carcass makes you temporarily recoil and reconsider the safety of delving into its entrails",
+      "The sputters of acid on your gear you earned by digging into this corpse could maybe warrant the use of some water to douse",
+      "Most cavities in its anatomy seem to have been filled with an acidic solution mixed with a foul black liquid and pieces of its slowly melting tissues",
+      "You are almost certain that its flesh should not be able to contain such a virulently aggressive liquid, yet, it does",
+      "You have to step back as a gland you inadvertently into seems to perform a neutralization reaction between its contents and the rest of its fluids"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_shadow_general>",
+    "text": [
+      "This corpse seems completely ordinary now, and dissection brings you no further to asserting what gave the creature its peculiar properties",
+      "The lingering shadows surrounding it make your work quite a lot harder as they seem to deliberately grow more opaque around your tool as if they wished to hide the monster's secrets",
+      "You spot a few pores still weakly sputtering thin puffs of black smoke, investigating them further reveals glands full of fuming black liquid linked to the pores with long tubes",
+      "You make one incision and the resulting cloud of black smoke forces you to take a short break as it becomes impossible to see anything",
+      "Despite it not being very evident from the outside, it seems the entire digestive tract of the abomination was bloated with its obscuring smoke"
```

Comment:
```suggestion
      "This corpse seems completely ordinary now, and dissection brings you no further to asserting what gave the creature its peculiar properties.",
      "The lingering shadows surrounding it make your work quite a lot harder as they seem to deliberately grow more opaque around your tool as if they wished to hide the monster's secrets.",
      "You spot a few pores still weakly sputtering thin puffs of black smoke, investigating them further reveals glands full of fuming black liquid linked to the pores with long tubes.",
      "You make one incision and the resulting cloud of black smoke forces you to take a short break as it becomes impossible to see anything.",
      "Despite it not being very evident from the outside, it seems the entire digestive tract of the abomination was bloated with its obscuring smoke."
```

Reply:
No end punctuation to add

Reply:
Got feedback from Venera, adding all your suggestions then removing punctuation from the composite snippets.